addons/l10n_it_edi_withholding/models/account_move.py

```diff
@@ -186,21 +189,57 @@ def _l10n_it_edi_get_extra_info(self, company, document_type, body_tree, incomin
                  + type_tax_use_domain),
                 vat_only=False)
             if pension_fund_tax:
-                pension_fund_taxes.append(pension_fund_tax)
+                pension_fund_taxes[vat_tax_factor_percent] = pension_fund_tax
```

Comment:
Is it possible that different pension fund taxes have the same rate?

Reply:
We don't handle multiple pension funds on the same invoice at the moment.
It would be extra complicated, with this mixture of global + line tags... 

A vendor is only subscribed to a pension fund, there *might* be invoices where different professionals belonging to different pension funds and to the same group (accountants and notaries for example) invoice together... but it's a very rare use case and we don't manage it for now. 

No one has asked for this and this is a "bugfix".